src/main/java/org/jvnet/hudson/test/TemporaryDirectoryAllocator.java

```diff
@@ -85,15 +85,25 @@ public synchronized File allocate() throws IOException {
      * Deletes all allocated temporary directories.
      */
     public synchronized void dispose() throws IOException, InterruptedException {
+        // TODO when we bump the Jenkins dependency to 2.157
```

Comment:
…yes?

Reply:
still on 2.60.3 as a minimum version?   (https://github.com/jenkinsci/jenkins-test-harness/blob/master/pom.xml#L87)

Reply:
Yes that is our minimum, I meant what do we do when the minimum is 2.157+?

Reply:
ahh.. I think that may be the issue we are working around here was introduced in 2.157

https://github.com/jenkinsci/jenkins/commit/3ee6a384849ed65c7fd1a0c7befee8491b6c72b2#diff-ac5ef183142d5d8905bacc2d534567d7

Reply:
Oh OK, JENKINS-60308 then.

I am looking into whether a plain directory deletion fares better in Windows CI.